src/cpp/rtps/writer/ReaderProxy.cpp

```diff
@@ -455,6 +455,19 @@ ReaderProxy::ChangeConstIterator ReaderProxy::find_change(const SequenceNumber_t
         : it->getSequenceNumber() == seq_num ? it : end;
 }
 
+bool ReaderProxy::are_there_gaps()
+{
+    bool are_there_gaps = false;
+
+    if (changes_for_reader_.size() > 0 &&
```

Comment:
We could just return the condition of this if

Reply:
Done